conjur/resource.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+"""
+Resource module
+"""
+
+# pylint: disable=too-few-public-methods
+class Resource:
+    """
+    Used for representing Conjur resources
+    """
+    def __init__(self, **arg_params):
```

Comment:
Why do you prefer it as dictionary instead of list of params?
```suggestion
    def __init__(self, type, name):
```

Reply:
no reason. Updating